UnityProject/Assets/Scripts/UI/UI Bottom/UI_ItemSlot.cs

```diff
@@ -145,7 +145,7 @@ public void SetSecondaryImage(Sprite sprite)
 	public GameObject Clear()
 	{
 		PlayerScript lps = PlayerManager.LocalPlayerScript;
-		if (!lps || lps.canNotInteract())
+		if (!lps)
```

Comment:
does it fix things like #1512?

Reply:
Unfortunately it doesn't seem relevant to #1512 since it still occurs. clear should be clearing the UI space when called and it was causing some issues when it's checking if the player can interact or not.